odxtools/units.py

```diff
@@ -0,0 +1,290 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021 MBition GmbH
```

Comment:
2022?!

Reply:
Good point! Should we changed that in all files actually?

Reply:
I guess only when they get modified. Anyway, I think the factual legal relevance of this comment is (close to?) zero anyway...